src/bitcoin-cli-res.rc

```diff
@@ -17,7 +17,7 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Zcash"
-            VALUE "FileDescription",    "zcash-cli (JSON-RPC client for Zcash)"
+            VALUE "FileDescription",    "zcash-cli (JSON-RPC client for " PACKAGE_NAME ")"
             VALUE "FileVersion",        VER_FILEVERSION_STR
             VALUE "InternalName",       "zcash-cli"
             VALUE "LegalCopyright",     COPYRIGHT_STR
```

Comment:
`COPYRIGHT_STR` is used here directly (not via `CopyrightHolders` function), so it's a regression in the Windows executable that it now omits attribution to the Bitcoin Core Developers.

Reply:
```suggestion
            VALUE "LegalCopyright",     COPYRIGHT_STR " " BITCOIN_COPYRIGHT_STR
```